packages/vue/examples/with-nuxt-typescript/pages/index.vue

```diff
@@ -0,0 +1,19 @@
+<script setup>
+import { getServerState } from '@appbaseio/reactivesearch-vue';
+import Search from '../components/search.vue';
+
+// eslint-disable-next-line no-undef
+const route = useRoute();
+// eslint-disable-next-line no-unused-vars,no-undef
```

Comment:
This can't be removed. useRoute is defined by nuxt compiler and hence would be undefined here. Also this is copied from the already existing nuxt example which had this rules disabled and converted into typescript.